quicktest_test.go

```diff
@@ -474,11 +474,17 @@ func (t *testingT) Parallel() {
 	t.parallel = true
 }
 
-// Fatal overrides *testing.T.Fatal so that messages are collected and the
-// goroutine is not killed.
+// Run overrides *testing.T.Run.
 func (t *testingT) Run(name string, f func(t *testing.T)) bool {
 	t.subTestName, t.subTestT = name, &testing.T{}
-	f(t.subTestT)
+	ch := make(chan struct{})
```

Comment:
// Run the subtest in its own goroutine so that if it calls runtime.GoExit,
    // we can still return appropriately.

?